.github/workflows/ci-nym-vpn-core-ios.yml

```diff
@@ -71,7 +72,7 @@ jobs:
             --language swift --out-dir build -n
 
       - name: Upload generated uniffi file
-        if: steps.diff_ios.outcome == 'failure'
+        if: steps.generate_uniffi.outcome == 'success'
```

Comment:
Not needed. If the above fails this wont run anyway

Reply:
Right. Thanks. Simply removed the bogus if condition